src/main/webapp/assets/modules/create-event/createEventController.js

```diff
@@ -1,20 +1,4 @@
-techlooper.controller("createEventController", function ($scope, $translate, jsonValue, apiService) {
-
-  var placeholder = $translate.instant('whoJoinAndWhyEx');
-  $('#txtWhyEvent').val(placeholder);
-  $('#txtWhyEvent').focus(function () {
-    if ($(this).val() === placeholder) {
-      $(this).val('');
-      $(this).removeClass('change-color');
-    }
-  });
-
-  $('#txtWhyEvent').blur(function () {
-    if ($(this).val() === '') {
-      $(this).val(placeholder);
-      $(this).addClass('change-color');
-    }
-  });
+techlooper.controller("createEventController", function ($scope, $translate, jsonValue, apiService, $rootScope) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'$rootScope' is defined but never used.